transformer_engine/common/recipe.py

```diff
@@ -98,6 +98,14 @@ def scaling_factor_compute(amax: Tensor,
     override_linear_precision: Tuple(bool, bool, bool), default=(False, False, False)
                               Whether or not the execute the `fprop`, `dgrad`, and `wgrad`
                               GEMMs (respectively) in higher precision when using FP8.
+    reduce_amax: bool, default = `True`
+                By default, if `torch.distributed` is initialized, the `amax` value for fp8
+                tensors is reduced across the specified `fp8_group` using the `fp8_autocast`
+                API after every iteration. This keeps the amaxes and scaling factors synced
+                across the given distributed group. If set to `False`, this reduction is
+                skipped and every GPU maintains localized amax and scaling factors. To ensure
```

Comment:
`local` instead of `localized`?

Reply:
Also, we should make it more clear that the following sentence is only about the `False` setting. Maybe something like this:
```
To ensure results are numerically identical across checkpointing boundaries in this case,
```
?